usage/test-framework/src/main/java/org/apache/brooklyn/test/framework/SimpleShellCommandTest.java

```diff
@@ -36,27 +40,10 @@
 @ImplementedBy(SimpleShellCommandTestImpl.class)
 public interface SimpleShellCommandTest extends BaseTest {
 
-    String TMP_DEFAULT = "/tmp";
-
-    /**
-     * Equals assertion on command result.
-     */
-    String EQUALS = "equals";
-
     /**
-     * String contains assertion on command result.
+     * Default location for temporary files.
      */
-    String CONTAINS = "contains";
-
-    /**
-     * Regex match assertion on command result.
-     */
-    String MATCHES = "matches";
-
-    /**
-     * Is-empty match assertion on command result.
-     */
-    String IS_EMPTY = "isEmpty";
+    String TMP_DEFAULT = "/tmp";
```

Comment:
You can use `org.apache.brooklyn.util.os.Os.tmp()`. But it doesn't matter given that this is just for tests. So consider this just information, rather than needing changed.